optimizely/decision/composite_experiment_service.go

```diff
@@ -44,6 +47,13 @@ func NewCompositeExperimentService() *CompositeExperimentService {
 
 // GetDecision returns a decision for the given experiment and user context
 func (s CompositeExperimentService) GetDecision(decisionContext ExperimentDecisionContext, userContext entities.UserContext) (ExperimentDecision, error) {
+
+	if decisionContext.Experiment.Status != entities.Running {
```

Comment:
In the new world, experiment status is no longer relevant. In fact we should probably deprecate the experiment status tests (i.e. run them in 2.x only) because we've changed the datafile builds to only including running experiments.